web/src/styles/base/flex.scss

```diff
@@ -92,6 +92,10 @@
   flex-direction: column-reverse;
 }
 
+.flex-column {
```

Comment:
this is already in this file as `.fdc`

Reply:
Not sure how this showed up in this story, b/c it already existed in the codebase & is used in various components.